src/Sulu/Bundle/AdminBundle/Resources/js/components/CircleSelection/CircleSelection.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+import type {Node} from 'react';
+import React from 'react';
+import withContainerSize from '../withContainerSize';
+import CircleSelectionRenderer from './CircleSelectionRenderer';
+import type {SelectionData} from './types';
+import circleSelectionStyles from './circleSelection.scss';
+
+type Props = {
+    children?: Node,
+    disabled: boolean,
+    filled: boolean,
+    label?: string,
+    maxRadius?: number,
+    minRadius?: number,
+    onChange: (value: ?SelectionData) => void,
+    percentageValues: boolean,
+    resizable: boolean,
+    round: boolean,
+    value: SelectionData | typeof undefined,
+};
+
+class CircleSelection extends React.Component<Props & {
+    containerHeight: number,
+    containerWidth: number,
```

Comment:
Because the default export of that file is a `CircleSelection` wrapped by `withContainerSize` ... The CircleSelection itself needs to know `containerWidth` and `containerHeight`, but those props are set by `withContainerSize`

Reply:
I think my main confusion stems from [the other comment](https://github.com/sulu/sulu/pull/5434#discussion_r494962531), let's continue our discussion there.